docs/HARDWARE.md

```diff
@@ -70,6 +71,14 @@ Once the reader discovery process starts yielding results, those results are pro
 
 When the user selects a reader to connect to, we pass that public model object back to the `CardReaderService`, via the `connect()` method. At that point, for the integration with the Stripe Terminal SDK, we will look into the internal cache of discovered readers, find the one that matches the serial number of the parameter provided, and attempt a connection with the StripeTerminal.CardReader found.
 
+### Disconnecting from a card reader
+
+There are to ways a client and a card reader can disconnect: either by a user-initiated action, or because the reader has gone offline.
+
+The disconnection is initiated by a user by just calling the `disconnect()` method in `CardReaderService` This is an async operation.
+
+Card Readers can shut down as well after a certain time being inactive. In that case, the integration with the Stripe Terminal SDK will catch the disconnection and publish a new array of connected readers (empty) via the `connectedReaders` publisher.
```

Comment:
> a new array of connected readers (empty)

Just noting that we might want to replace this array with an optional and we should update the documentation if we change that.